### ISSUE TITLE
Question regarding figure 5 on the paper

### ISSUE THREAD

**Issue by ajboloor** [facebookresearch/mae#163]
What pipeline was used to report the finetuned accuracy numbers on fig. 5 of the paper?

Specifically, when is the masking done?
Is it done only during pretraining? 

Or is it done also during finetuning?
